lib/mock-axios.ts

```diff
@@ -179,5 +181,11 @@ MockAxios.reset = () => {
   MockAxios.all.mockClear();
 };
 
+MockAxios.Cancel = Cancel;
+MockAxios.CancelToken = CancelToken;

```

Comment:
This now results (after my changes...) in an TS Error:

```
lib/mock-axios.ts:185:1 - error TS2322: Type 'typeof CancelToken' is not assignable to type 'CancelTokenStatic'.
  Types of property 'source' are incompatible.
    Type '() => { cancel: Canceler; token: CancelToken; }' is not assignable to type '() => CancelTokenSource'.
      Type '{ cancel: Canceler; token: CancelToken; }' is not assignable to type 'CancelTokenSource'.
        Types of property 'token' are incompatible.
          Property 'promise' is missing in type 'CancelToken' but required in type 'CancelToken'.
```

as the provided CancelToken lacks a `promise`-field which is required by the matching interface in `mock-axios-types.ts`

Reply:
I did not implement the promise part, as it's not needed unless we go for an actual implementation (it's not publicly exposed either).

Reply:
Fixed in https://github.com/VanTanev/jest-mock-axios/commit/7481b9f9a3be0a92168541deda16d251184027e0

I am happy to duplicate the definition of `Canceller` type to avoid exporting it from axios-mock-types, which are public-facing.